jobs/rocoto/ocnanalecen.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+###############################################################
+# Source UFSDA workflow modules
+. "${HOMEgfs}/ush/load_ufsda_modules.sh"
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+export job="ocnanalecen"
+export jobid="${job}.$$"
+
+###############################################################
+# Setup Python path for GDASApp ush
+PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
```

Comment:
If `PYTHONPATH` is not defined before this point, the `PYTHONPATH` set after this line will end in a `:`.  This should be rectified.

Reply:
fixed